package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gqlint",
-  "version": "1.6.1",
+  "version": "1.8.0",
```

Comment:
Please rebase on master (git pull --rebase origin master) as I don't think you have the latest code. I'll do the bumping :)